arcgis-ios-sdk-samples/Maps/Open mobile map (map package)/README.md

```diff
@@ -1,14 +1,32 @@
-# Open mobile map (map package)
+# Open mobile map package
 
-This sample demonstrates how to open a mobile map from a map package
+Display a map from a mobile map package.
 
-![](image1.png)
+![Image of open mobile map package](open-mobile-map-package.png)
+
+## Use case
+
+An .mmpk file is an archive containing the data (specifically, basemaps and features) used to display an offline map.
```

Comment:
How do people read `.mmpk` - Should it be a or an? 🤯

Reply:
I think either one is fine. But now that you mention it, it probably should be _a_. _A_ dot mmpk file.